src/main/reservation/main/G1HRPS/ReservationManager.java

```diff
@@ -53,21 +70,28 @@ public void SaveDB() {
 	 * @param new_status
 	 */
 	public boolean SetReservationStatus(Reservation reservation, ReservationStatus new_status) {
```

Comment:
To remove SetReservationStatus and GetReservationStatus as these are already implemented by Reservation.GetStatus and Reservation.SetStatus.